tensorflow/training/docker/1.15/py36/Dockerfile.cpu

```diff
@@ -125,7 +125,7 @@ RUN pip install --no-cache-dir -U \
     keras_preprocessing==1.1.0 \
     keras==2.3.1 \
     requests==2.22.0 \
-    "sagemaker<2" \
+    "sagemaker<3" \
```

Comment:
nit: Could you set a minimum version here as well?

I've been using `sagemaker>=2,<3` to enforce that the version installed is always `sagemaker==2.x`.